src/framework/chain_normalizers.ini

```diff
@@ -14,6 +14,7 @@ rescaler=movit.resample,swscale,gtkrescale,rescale
 resizer=movit.resize,resize
 movitconvert=movit.convert
 imageconvert=avcolor_space,imageconvert
+colorspace=colorspace
```

Comment:
Anything that involves pixel blending or interpolation should be done in linear color if that is requested. Is it possible to move to first in the list since `deinterlace` and `rescaler` does that? I have not looked into it yet, but this `colorspace` conversion should be skipped when movit is being used because it already does that and usually faster (GPU). Don't worry about the deinterlace case with respect to movit; it has that but I need to integrate it.